workloads/kube-burner-ocp-wrapper/run.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash -e
+
+set -e
+
+hypershift(){
+  if [[ -z ${MC_KUBECONFIG} ]]; then
+    echo "MC_KUBECONFIG variable must be defined"
+    exit 1
+  fi
+  
+  HC_ID=$(oc get infrastructure cluster -o go-template --template='{{.status.infrastructureName}}')
+  HC_NAME=$(oc get infrastructure cluster -o go-template --template='{{range .status.platformStatus.aws.resourceTags}}{{if eq .key "api.openshift.com/name" }}{{.value}}{{end}}{{end}}')
+  
+  
+  if [[ -z ${HC_ID} ]] || [[ -z ${HC_NAME} ]]; then
+    echo "Couldn't obtain hosted cluster id and/or hosted cluster name"
+    echo -e "HC_ID: ${HC_ID}\nHC_NAME: ${HC_NAME}"
+    exit 1
+  fi
+  
+  # Hosted control-plane namespace is composed by the cluster ID plus the cluster name
+  HCP_NAMESPACE=${HC_ID}-${HC_NAME}
+  
+  echo "Creating OBO route"
+  oc --kubeconfig=${MC_KUBECONFIG} apply -f - << EOF
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app.kubernetes.io/instance: hypershift-monitoring-stack
+  name: prometheus-hypershift-monitoring-stack
+  namespace: openshift-observability-operator
+spec:
+  ports:
+  - port: 9090
+    protocol: TCP
+    targetPort: web
+  selector:
+    app.kubernetes.io/instance: hypershift-monitoring-stack
+---
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  name: prometheus-hypershift
+  namespace: openshift-observability-operator
+spec:
+  port:
+    targetPort: 9090
+  to:
+    name: prometheus-hypershift-monitoring-stack
+EOF
+  echo "Fetching OBO endpoint"
+  MC_OBO=http://$(oc --kubeconfig=${MC_KUBECONFIG} get route -n openshift-observability-operator prometheus-hypershift -o jsonpath="{.spec.host}")
+  MC_PROMETHEUS=https://$(oc --kubeconfig=${MC_KUBECONFIG} get route -n openshift-monitoring prometheus-k8s -o jsonpath="{.spec.host}")
+  MC_PROMETHEUS_TOKEN=$(oc --kubeconfig=${MC_KUBECONFIG} sa new-token -n openshift-monitoring prometheus-k8s)
+  HOSTED_PROMETHEUS=https://$(oc get route -n openshift-monitoring prometheus-k8s -o jsonpath="{.spec.host}")
+  HOSTED_PROMETHEUS_TOKEN=$(oc sa new-token -n openshift-monitoring prometheus-k8s)
+  echo "Exporting required vars"
+  cat << EOF
```

Comment:
I think we can we reduce this statement and the below exports into a single for loop, looping over all the variables and does both echoing and exporting of variables.

Reply:
Not sure how to do it without using `eval`, do you have any suggestion?

Reply:
Sure. Here is a for loop for doing both echo and exports below. Thank you!
```
echo "Exporting required vars"
vars=(MC_OBO MC_PROMETHEUS MC_PROMETHEUS_TOKEN HOSTED_PROMETHEUS HOSTED_PROMETHEUS_TOKEN HCP_NAMESPACE)
for var in "${vars[@]}"; do
  value="${!var}"
  echo "${var}: ${value}"
  export "${var}=${value}"
done
```

Reply:
Thanks!, I think I'll stick with the current implementation, that code is a bit hacky and difficult to read...